gui/create-item.lua

```diff
@@ -201,6 +400,21 @@ function hackWish(unit)
    if not creatureok then return end
    mattype,matindex=getCreatureRaceAndCaste(creatureTable[3])
   end
+  if df.item_type[itemtype]=='CORPSEPIECE' then
+	bodpartok,bodypart=script.showListPrompt('Wish','What body part should it be?',COLOR_LIGHTGREEN,getCreaturePartList(mattype,matindex))
```

Comment:
Use `local` to prevent these from becoming global variables:
```suggestion
	local bodpartok,bodypart=script.showListPrompt('Wish','What body part should it be?',COLOR_LIGHTGREEN,getCreaturePartList(mattype,matindex))
```

Reply:
'bodypart' is referenced by a following 'createCorpsePiece()', but i made `bodypart` local to hackWish()